docs/docs/tutorial.md

```diff
@@ -21,7 +21,7 @@ It'll also have a few neat features:
 
 ### Want to skip all this and just see the source?
 
-[It's all on GitHub.](https://github.com/reactjs/react-tutorial)
+[It's all on GitHub](https://github.com/reactjs/react-tutorial). Or, [view and edit your own live copy on HyperDev](https://hyperdev.com/#!/remix/react_tutorial/0539a08d-bcda-458e-8647-94813e4248b4).
```

Comment:
So, not the latest version of the *repo* but the latest version of that hyperdev project (which is a snapshot of the repo and is another thing to maintain when we make changes…). To that point, it's already out of date with changes we made last week :/

Based on http://support.hyperweb.space/t/keep-github-repo-from-packages-json-in-sync-with-code/221, it looks like the ability to push to a repo directly and have a project updated is in the works. Let's hold off until that feature is available. We've heard very few comments about having to run a local server to do the tutorial so it doesn't seem like it's a pain point (but I would love to lower the barrier even further)